internal/tiltfile/config/config_def.go

```diff
@@ -119,7 +120,9 @@ func (cd ConfigDef) parseArgs(args []string) (ret configMap, output string, err
 
 	if len(fs.Args()) > 0 {
 		if cd.positionalSettingName == "" {
-			return nil, w.String(), errors.New("positional args were specified, but none were expected (no setting defined with args=True)")
+			return nil, w.String(), fmt.Errorf(
+				"invalid tiltfile config args: positional CLI args (%q) were specified, but none were expected.\n"+
+					"See https://docs.tilt.dev/tiltfile_config.html#positional-arguments for examples.", strings.Join(fs.Args(), " "))
```

Comment:
wdyt of taking the prefixes out of `parseArgs` and adding an `error.Wrap(err, "invalid tilitfile config args:")` at the call site in `incorporateArgs`?

(parseArgs` has three places that can return errors, and now two of them have (differing) prefixes)

Reply:
Good call - see latest (b2fb211fe) for changes

### parse error
formatting has been improved here - before you got the error + usage as INFO (not ERROR) and then the error again (w/o usage) as ERROR; now everything is 1x in ERROR
<img width="668" alt="arg parse error" src="https://user-images.githubusercontent.com/841263/133448270-ec82380b-c8c3-4786-9f37-4900138165e7.png">

### unexpected positional args
<img width="1047" alt="unexpected positional args error" src="https://user-images.githubusercontent.com/841263/133448747-3d443a9c-0031-41a8-ab99-167c92a05fb5.png">

### invalid positional args
(bonus: added a missing `)` on this error message)
<img width="1039" alt="invalid positional args error" src="https://user-images.githubusercontent.com/841263/133448630-03070bc5-dca7-4b45-bc2f-a7a5b5235756.png">